lms/djangoapps/course_api/blocks/transformers/video_stream_priority.py

```diff
@@ -66,6 +69,6 @@ def transform(self, usage_info, block_structure):
             encoded_videos = student_view_data.get('encoded_videos')
             for video_format, video_data in encoded_videos.items():
                 if DEPRECATE_YOUTUBE.is_enabled(usage_info.course_key):
-                    video_data['stream_priority'] = self.DEPRECATE_YOUTUBE_VIDEO_STREAM_PRIORITY[video_format]
+                    video_data['stream_priority'] = self.DEPRECATE_YOUTUBE_VIDEO_STREAM_PRIORITY.get(video_format, -1)
```

Comment:
is -1 a valid value? Is that going to cause any problems downstream?

Reply:
As of now the field `stream_priority` is not being used. Implementation in mobile apps is to be made after this change in backend. In this context, `-1` can be communicated as a  default value for edge cases and can be dealt accordingly.

Reply:
Excellent. Thanks